internal/pod_models.go

```diff
@@ -426,7 +430,7 @@ func GetPodSpec(cluster *fdbv1beta2.FoundationDBCluster, processGroup *fdbv1beta
 
 	podName := processGroup.GetPodName(cluster)
 	if useUnifiedImage {
-		err = configureContainersForUnifiedImages(cluster, mainContainer, sidecarContainer, processGroup, desiredVersion)
+		err = configureContainersForUnifiedImages(cluster, mainContainer, sidecarContainer, processGroup, desiredVersion, logGroup)
```

Comment:
I would prefer if we move the method above (the one where we get the log group), into a `FoundationDBCluster` method and use that instead of adding another parameter. What are your thoughts on that?

Reply:
Like `cluster.GetLogGroup()`? yes I completely agree

Reply:
done